lib/hooks/update/index.js

```diff
@@ -0,0 +1,32 @@
+const { get, set } = require('lodash');
+const { BadRequestError } = require('@asl/service/errors');
+
+const updateModel = (model) => {
+  const payload = get(model, 'meta.payload', {});
+  const updates = get(model, 'data');
```

Comment:
this is a bit confusing here cause `updates` will be the existing model data at this point won't it?

What about something like 

```
const data = get(model, 'data');
const updates = {};

if (payload.data) {
  updates.data = payload.data;
}

if (payload.meta) {
  updates.meta = payload.meta;
}

return model.patch(merge(data, updates));
```

Reply:
sure, if you use the handy insert suggestion, then I just have to click the button :)

Reply:
actually that won't work cos you chucked a merge in there, updating